README.md

```diff
@@ -1 +1,115 @@
 # OKRs 2018 Q4
+
+## Bring 'Code as Data' to market with enterprises
+
+### Ensure gitbase interoperability with 3rd party tools [DR]
+
+- [ ] Drivers for in-house languages (Python, Go, Java).
+- [ ] mysqldump
+- [ ] grafana
+- [ ] Tableau
+- [ ] Drivers for other popular languages (JavaScript, Ruby, .NET, PHP).
+
+### Get gitbase enterprise ready [DR]
+
+- [ ] Authentication and authorization abstractions prepared for future integration with LDAP and Kerberos
```

Comment:
`authorization abstractions` means an authorization to specific repositories, per example?

Reply:
No. Just to connect to gitbase and execute `read` (SELECT) or `write` (CREATE INDEX) queries.